deployment/helm/ditto/values.yaml

```diff
@@ -217,14 +217,12 @@ ingress:
         backendSuffix: gateway
       - path: /health
         backendSuffix: gateway
+    kubernetesAuthAnnotations: |
+      nginx.ingress.kubernetes.io/auth-type: basic
+      nginx.ingress.kubernetes.io/auth-secret: {{ .Release.Name }}-nginx-ingress-htpasswd
```

Comment:
I don't think that it is possible to use Helm variables in the `values.yaml` ..
Are you sure about that?

Reply:
Ah, wow .. did not see that.
That is a nice hack, thx for explaining 👍